wordpress-mobile/WordPress-iOS
# Stats: fix stats refresh when day changes in background

This PR fixes an issue where stats were not updating correctly when the date incremented on iOS in the background

Fixes #19688

To test:
1. Set iPhone clock to yesterday on iOS via Settings
2. Fresh install JetPack app and login to account that has stats
3. Open Stats
4. Tap the Days tab
5. Background the JetPack app
6. Set iPhone clock back to today on iOS via Settings
7. Relaunch the JetPack app which should still be on the Stats -> Days tab screen
8. Notice that yesterday's date is selected
9. Select today by clicking ">" icon
10. Notice that now the "Days" tab displays today's data correctly

## Regression Notes
1. Potential unintended areas of impact
Stats

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
